### PR TITLE
add doc details

### PR DESCRIPTION
**Pull request by alex102400**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

补充文档细节

### Full changelog

在文档中添加第三方依赖的链接和检查命令等说明

### Issues resolved

Fix #XXX

### CODE REVIEW

File: doc/how-to-build-cn.md [hunk 1]
```diff
@@ -122,7 +123,13 @@ Makefile rules:
 
 ## 5. 更新 Admin API 的 token ，保护 Apache APISIX
 
-修改 `conf/config.yaml` 中的 `apisix.admin_key` 并重启服务。例如下面例子：
+修改 `conf/config.yaml` 中的 `apisix.admin_key` 
+重启服务
```
Comment (juzhiyuan) [inline doc/how-to-build-cn.md:127]: 修改 `conf/config.yaml` 中的 `apisix.admin_key` 并重启服务：

File: README_CN.md [hunk 2]
```diff
@@ -145,14 +145,21 @@ sudo apisix start
...
+```
 
 2. 下载 [Dashboard](https://github.com/apache/incubator-apisix-dashboard) 的源码：
+RPM/yum安装的apisix-1.2版本，默认已安装至/usr/local/apisix/dashboard，参考第5步配置允许访问即可。
 ```
 git clone https://github.com/apache/incubator-apisix-dashboard.git
 ```
 
 3. 安装 [yarn](https://yarnpkg.com/zh-Hans/docs/install)
+```
```
Comment (juzhiyuan) [inline README_CN.md:160]: bash or shell

File: README_CN.md [hunk 3]
```diff
@@ -145,14 +145,21 @@ sudo apisix start
 
 APISIX 内置了对 Dashboard 的支持，使用步骤如下：
 
-1. 确保你的运行环境中使用了最新的 Node.js 版本（>= 10）。
+1. 确保你的运行环境中使用了最新的 [Node.js](https://nodejs.org/) 版本（>= 10）。
+```
```
Comment (juzhiyuan) [inline README_CN.md:149]: bash or shell

**Comment by moonming** [comment]
@alex102400 I just updated the dashboard build steps, please rebase master, thx

**Comment by moonming** [comment]
@alex102400 any news? I will close this PR first. You can reopen it if need.

**Pull request closed without merging**